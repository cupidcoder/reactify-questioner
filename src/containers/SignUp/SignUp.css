.SignUp {
  min-height: 80vh;
}