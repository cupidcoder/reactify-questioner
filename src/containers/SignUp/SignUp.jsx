import React from 'react';
import { SignUpFormComponent } from '../../components/SignUpForm/SignUpForm';
import style from './SignUp.css';

const SignUp = () => (
  <div className={style.SignUp}>
    <SignUpFormComponent />
  </div>
);

export default SignUp;
