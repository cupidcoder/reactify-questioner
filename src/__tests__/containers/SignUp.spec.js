import React from 'react';
import { shallow } from 'enzyme';
import SignUp from '../../containers/SignUp/SignUp';

describe('SignUp', () => {
  it('Should correctly save snapshot of the SignUp page', () => {
    const wrapper = shallow(<SignUp />);
    expect(wrapper).toMatchSnapshot();
  });
});
