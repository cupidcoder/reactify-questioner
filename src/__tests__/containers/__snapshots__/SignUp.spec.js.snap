// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp Should correctly save snapshot of the SignUp page 1`] = `ShallowWrapper {}`;
